services/apps/alcs/src/alcs/search/search.dto.ts

```diff
@@ -16,6 +16,7 @@ export class SearchResultDto {
   fileNumber: string;
   boardCode?: string;
   label?: ApplicationTypeDto;
+  outcome?: string
```

Comment:
I don't think you updated the ALCS search, only the portal public search, this should be undone.

Reply:
ticket just mentions public search